scrape_books/spiders/books.py

```diff
@@ -0,0 +1,63 @@
+from urllib.parse import urljoin
+
+import scrapy
+from scrapy.http import Response
+
+
+class BooksSpider(scrapy.Spider):
+    name = "books"
+    allowed_domains = ["books.toscrape.com"]
+    start_urls = ["https://books.toscrape.com/"]
```

Comment:
should be constant

Reply:
if I write it as a constant, my program crashes

Reply:
![image](https://user-images.githubusercontent.com/93856423/194248975-e7c56a92-23a9-4f18-869f-5de1f382f974.png)

Reply:
🥲🥲🥲🥲🥲🥲